KataClub/Logic/ArithmeticProgressionView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ArithmeticProgressionView.swift
+//  KataClub
+//
+//  Created by Tom Phillips on 1/27/22.
+//
+
+import SwiftUI
+
+struct ArithmeticProgressionView: View {
+    @State private var inputArrayText = ""
+    @State private var output = ""
+    
+    @State var inputArray = [Int]()
+    
+    var body: some View {
+        ScrollView {
+            VStack{
+                Text("Find the Missing Element of an Arithmetic Progression")
+                    .font(.largeTitle)
+                    .padding()
+                
+                Text("""
+                An Arithmetic Progression is defined as one in which there is a constant difference between the consecutive terms of a given series of numbers. For example, in the sequence [1, 2, 3, 4] the arithmetic progression is 1 and in [1,3,5,7], the arithmetic progression is 2.
+                
+                You are provided with consecutive elements of an Arithmetic Progression. There is however one hitch: exactly one term from the original series is missing from the set of numbers which have been given to you. The rest of the given series is the same as the original Arithmetic Progression. Find the missing term. If the sequence has no missing term, return 'nil' instead.
```

Comment:
is it confusing to put nil in single quotes? I just don't want people to think they should return a string with the word "nil"